0.download_data/jump_dataset.csv

```diff
@@ -0,0 +1,51 @@
+source_4,2020_11_04_CPJUMP1,BR00116991,s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00116991/BR00116991.sqlite,COMPOUND
```

Comment:
what is the provenance of this file? (How was this file generated?)

You can specify this provenance in a README :)

Reply:
Also, are these all `COMPOUND` plates? Are there other `CRISPR` and `ORF`?

Reply:
These include all of the plates. I could also include the code for generating this file, but it essentially just creates the s3 links with the plate names and the aws file path.

Reply:
Adding code that generates this file is important for reproducibility :)

It is good practice to also include details on how you executed the file (e.g., explicit run-time instructions)

Reply:
Sounds good, just added this!